src/UI/Art/SpriteArtist/SpriteLibrary.java

```diff
@@ -154,99 +154,50 @@ private static void createTerrainSpriteSet(SpriteSetKey spriteKey)
-    }
     else if( terrainType == TerrainType.GRASS )
     {
-      ss = new TerrainSpriteSet(terrainType, formatString, w, h);
       ss.addTileTransition(TerrainType.BRIDGE, String.format(transitionFormatString, terrainType.toString().toLowerCase(), "%s", "%s"), w, h);
     }
-    else if( terrainType == TerrainType.HEADQUARTERS )
-    {
-      ss = new TerrainSpriteSet(terrainType, formatString, w * 2, h * 2);
-    }
-    else if( terrainType == TerrainType.LAB )
-    {
-      // TODO: get actual sprites for this
-      ss = new TerrainSpriteSet(terrainType, formatString, w * 2, h * 2);
-    }
-    else if( terrainType == TerrainType.MOUNTAIN )
-    {
-      ss = new TerrainSpriteSet(terrainType, formatString, w * 2, h * 2);
-    }
     else if( terrainType == TerrainType.SEA )
     {
-      ss = new TerrainSpriteSet(terrainType, formatString, w, h);
       ss.addTileTransition(TerrainType.GRASS, String.format(transitionFormatString, terrainType.toString().toLowerCase(), "%s", "%s"), w, h);
       ss.addTileTransition(TerrainType.BRIDGE, String.format(transitionFormatString, terrainType.toString().toLowerCase(), "%s", "%s"), w, h);
       ss.addTileTransition(TerrainType.RIVER, String.format(transitionFormatString, terrainType.toString().toLowerCase(), "%s", "%s"), w, h);
     }
     else if( terrainType == TerrainType.REEF )
     {
-      ss = new TerrainSpriteSet(terrainType, formatString, w, h);
       ss.addTileTransition(TerrainType.SEA, String.format(transitionFormatString, terrainType.toString().toLowerCase(), "%s", "%s"), w, h);
     }
     else if( terrainType == TerrainType.RIVER )
     {
-      ss = new TerrainSpriteSet(terrainType, formatString, w, h);
       ss.addTileTransition(TerrainType.BRIDGE, String.format(transitionFormatString, terrainType.toString().toLowerCase(), "%s", "%s"), w, h);
       ss.addTerrainAffinity(TerrainType.SHOAL);
       ss.addTerrainAffinity(TerrainType.SEA);
     }
     else if( terrainType == TerrainType.ROAD )
     {
-      ss = new TerrainSpriteSet(terrainType, formatString, w, h);
       ss.addTerrainAffinity(TerrainType.BRIDGE);
       ss.addTerrainAffinity(TerrainType.HEADQUARTERS);
       ss.addTerrainAffinity(TerrainType.FACTORY);
     }
     else if( terrainType == TerrainType.SHOAL )
     {
-      ss = new TerrainSpriteSet(terrainType, formatString, w, h);
       ss.addTileTransition(TerrainType.SEA, String.format(transitionFormatString, terrainType.toString().toLowerCase(), "%s", "%s"), w, h);
       ss.addTileTransition(TerrainType.GRASS, String.format(transitionFormatString, terrainType.toString().toLowerCase(), "%s", "%s"), w, h);
       ss.addTileTransition(TerrainType.BRIDGE, String.format(transitionFormatString, terrainType.toString().toLowerCase(), "%s", "%s"), w, h);
       ss.addTileTransition(TerrainType.RIVER, String.format(transitionFormatString, terrainType.toString().toLowerCase(), "%s", "%s"), w, h);
     }
-    else
```

Comment:
Why is this unnecessary?

Reply:
It happens frequently, since there is no longer a case for every individual terrain type - the `TerrainSpriteSet` is generated unconditionally up above, so the "error" being reported is pretty irrelevant anyhow.